echopype/tests/visualize/test_plot.py

```diff
@@ -224,21 +224,25 @@ def test_water_level_echodata(water_level, expect_warning):
         ek_waveform_mode=range_kwargs.get('waveform_mode', 'CW'),
         ek_encode_mode=range_kwargs.get('encode_mode', 'power'),
     )
-    single_array = range_in_meter.isel(frequency=0, ping_time=0).values
+
+    single_array = range_in_meter.sel(channel='GPT  18 kHz 009072058c8d 1-1 ES18-11',
+                                      ping_time='2017-07-19T21:13:47.984999936').values
     no_input_water_level = False
+
     if isinstance(water_level, list):
         water_level = water_level[0]
         echodata.platform = echodata.platform.drop_vars('water_level')
         no_input_water_level = True
 
     if isinstance(water_level, xr.DataArray):
-        if 'frequency' in water_level.dims:
-            original_array = single_array + water_level.isel(frequency=0).values
+        if 'channel' in water_level.dims:
+            original_array = single_array + water_level.isel(channel=0).values
```

Comment:
For consistency, it seems like the `isel(channel=0)` selection could be replaced with what's done in line 244, `sel(channel='GPT 18 kHz 009072058c8d 1-1 ES18-11')`